2019.09/apache/entrypoint.sh

```diff
@@ -105,6 +105,8 @@ setup_ssmtp() {
 	fi
 }
 
+run_as "echo 'test';"
```

Comment:
Next time please remove the debug output before self-merging.

Reply:
yep, removed it with a followup Hotfix .. But this was needed because if anybody has pulled the latest files, the startup would have failed ..

Reply:
The debug output was needed or the rest of the changes?

Reply:
The rest was needed.. The output was just for testing.. And in case I had waited, travis would have nagged that there was an unwanted output..